airbyte-integrations/connectors-performance/destination-harness/src/main/java/io/airbyte/integrations/destination_performance/Main.java

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.integrations.destination_performance;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.airbyte.commons.io.IOs;
+import io.airbyte.commons.json.Jsons;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+
+@Slf4j
+public class Main {
+
+  private static final String CREDENTIALS_PATH = "secrets/%s_credentials.json";
+
+  public static void main(final String[] args) {
+    log.info("args: {}", Arrays.toString(args));
+    String image = null;
+    String dataset = "1m";
+
+    switch (args.length) {
+      case 1 -> image = args[0];
+      case 2 -> {
+        image = args[0];
+        dataset = args[1];
+      }
```

Comment:
best practice to include a default section in a switch to handle unexpected values gracefully